### ISSUE TITLE
The registration page in kindle for PC with accessibility plug in is not accessible.

### ISSUE THREAD

**Issue by sumandogra** [nvaccess/nvda#6486]
Steps to reproduce
1.	Open kindle for PC with accessibility plug-in.
2.	Go to tools and then options menu command.
3.	Go to registrations option and then TAB to register button.
4.	Press ENTER on the register button.
5.	Now try to navigate on the page that opens to register that has a combo box to select the relevant website, username and the password.

NVDA does not read through the page.  and without registration it is not possible to download the bought books from amazon.

**Comment by jcsteh** [comment]
Thanks for reporting. A major accessibility overhaul is in development for Kindle on Windows, though we haven't been given any dates on when it will be released to the public. Closing as invalid, since all issues related to Kindle will need to be completely re-evaluated.
